lab_7_llm/main.py

```diff
@@ -193,3 +274,11 @@ def run(self) -> dict | None:
         Returns:
             dict | None: A dictionary containing information about the calculated metric
         """
+        data = pd.read_csv(self._data_path).drop(columns=['Unnamed: 0'])
+
+        for metric in self._metrics:
+            if metric.value == "accuracy":
+                accuracy_metric = load(metric.value).compute(references=data['target'],
+                                                             predictions=data['predictions'])
+                return dict(accuracy_metric)
+        return None
```

Comment:
Delete.  Python function returns by default `None`

Reply:
Mypy checks made me do this, but now I've changed this function

done